src/commands/java_isopod_deploy.yml

```diff
@@ -0,0 +1,33 @@
+description: >
+  Deploy a java app.
+  Supports app-module from monorepo as well as single app from single-app-repo.
+  Expects an isopod.yml file to be present in the app directory (maven module directory for monorepo app, or project root for single-app-repo).
```

Comment:
Why this would be Java specific? For deploying only `isopod.yaml` and branch is important. It is stack agnostic

Reply:
i guess it can be inlined into the java_isopod_deploy job